modules/administration/pages/reports.adoc

```diff
@@ -0,0 +1,63 @@
+[[reports]]
+= Generate Reports
+
+The [command]``spacewalk-report`` command is used to produce a variety of reports.
+These reports can be helpful for taking inventory of your entitlements, subscribed systems, users, and organizations.
+Using reports is often simpler than gathering information manually from the {susemgr} {webui}, especially if you have many systems under management.
+
+To generate reports, you must have the [package]``spacewalk-reports`` package installed.
+
+The [command]``spacewalk-report`` command allows you to organize and display reports about content, systems, and user resources across {productname}.
+
+You can generate reports on:
+
+. System Inventory: lists all of the systems registered to {productname}.
+. Entitlements: lists all organizations on {productname}, sorted by system or channel entitlements.
+. Patches: lists all the patches relevant to the registered systems and sorts patches by severity, as well as the systems that apply to a particular patch.
+. Users: lists all the users registered to {productname} and any systems associated with a particular user.
+
+To get the report in CSV format, run this command at the command prompt on the server:
+
+----
+spacewalk-report <report_name>
+----
+
+This table lists the available reports:
```

Comment:
What about normalizing the descriptions?  We have "List of" and "Lists".  I'd prefer "List of".

Reply:
I removed all the "lists" and "lists of", it seems kinda redundant.